iOS Example Frame Carthage/iOS Example Frame Carthage.xcodeproj/project.pbxproj

```diff
@@ -21,9 +21,22 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
+		0B194AFC28A1132B001AEAFD /* UIFont+SFMono.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0B194AED28A1132B001AEAFD /* UIFont+SFMono.swift */; };
```

Comment:
Do we need to add them to pod example as well ?

Reply:
🤔 good question. Not being in use for now its hard to check, so I think it should be left for the CustomStyle2 refresh task when we'll use it and we may find that yes, it is needed :). For now I'd leave it as it is, since nothing is failing and pipeline is definitely compiling all distribution formats and demo projects.